Team broker schema agent

hardillb opened the pull request:
closes #5951
## Description

<!-- Describe your changes in detail -->
Adds support for Team Broker to MQTT Schema capture.



## Related Issue(s)

<!-- What issue does this PR relate to? -->
https://github.com/FlowFuse/flowfuse/issues/5951

## Checklist

<!-- https://flowfuse.com/handbook/development/#defining-done -->

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass <!-- If not adding test coverage, please clarify why not? -->
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production
 - [ ] Link to Changelog Entry PR, or note why one is not needed.

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

knolleary reviewed (commented):
Tested with localfs and all working. My main feedback is on the UX side.

There is an inconsistency between the team and 3rd party broker UX now that I think should be aligned.

Team broker has a button to start schema capture. Once clicked, there's no feedback (other than the button is now disabled) - and no way to stop the schema capture.

<img width="628" height="130" alt="image" src="https://github.com/user-attachments/assets/72c845c7-3138-4467-843e-46fdfde51043" />

3rd Party broker already has a toggle button for the same concept:

<img width="964" height="77" alt="image" src="https://github.com/user-attachments/assets/8e8ce861-7d1a-47f6-a7dc-f9c56d52ccd4" />

I think this needs to be aligned.

joepavitt commented:
Just having a think over on this. For what is meant to be a page showing _live_ data, it doesn't feel very _alive_. For Third-Party view, it's a little better with the green "Connected" pill, I think we should have an equivalent for the Team Broker too.

### Toggles

I see there being three positions where toggles _could_ live, shown here:

<img width="1670" height="501" alt="Screenshot 2025-09-17 at 13 32 26" src="https://github.com/user-attachments/assets/158c8866-40df-4576-8521-7f993b05ca5e" />

For the Team Broker, there is no value is a global "disconnect" option at the top-level, the left-side toggle for the hierarchy is _always_ on, and the right-side can be toggled on to watch payload schemas.

For Third-Party brokers, there is only a single connect/disconnect option, when that is connected then both the hierarchy and the topic are actively monitored.

I want to make sure we are communicating value here, and that it's clear to the user that there is _live_ data being gathered

hardillb commented:
@joepavitt I think I follow what your saying and agree, but if you could start on the front end and shout if there any backend changes needed

joepavitt commented:
@hardillb I've updated the UI, I've not implemented the _turning off_ the agent for Team Broker, but do feel free to if you think it'd be useful.

hardillb commented:
I've fixed the lint and removed the alert because it was triggering if you navigated from a disabled broker to the already running Team Broker.

I'm going to leave being able to turn team broker collection off manually for now

But it looks a lot more consistent across the 2 modes now.

knolleary commented:
Tested locally with driver-localfs.

knolleary commented:
@hardillb I have approved the core pr, driver-localfs and the agent. Due to their interdependencies when it comes to deploying to prod, will let you merge in the appropriate order.

hardillb commented:
@knolleary Thanks, will get with Steve today to wrap up the driver-docker side and the it can all be merged
